guides/common/modules/proc_extracting-gpg-public-key-fingerprints-from-release-files.adoc

```diff
@@ -1,7 +1,7 @@
 [id="Extracting_GPG_Public_Key_Fingerprints_from_Release_Files_{context}"]
 = Extracting GPG public key fingerprints from Release files
```

Comment:
```suggestion
= Extracting GPG public keys from Release files
```

Reply:
I did not change this because you don't really extract the GPG public key, but only its fingerprint from the "Release" file by "trial and error".

Reply:
In that case, I recommend adding "fingerprints" to the intro and procedure. It looks odd when we mention something in the title and then never refer to it again.